scarb/tests/build.rs

```diff
@@ -69,7 +69,7 @@ fn compile_with_syntax_error() {
         .build(&t);
 
     Scarb::quick_snapbox()
-        .arg("build")
+        .arg("build") // Todo(#137) check
```

Comment:
Could you please use the standard pattern for todo comments that we use elsewhere? :)